client/my-sites/upgrade-nudge/index.jsx

```diff
@@ -33,7 +31,8 @@ export default React.createClass( {
 		jetpack: React.PropTypes.bool,
 		compact: React.PropTypes.bool,
 		feature: React.PropTypes.oneOf( [ false, ...getValidFeatureKeys() ] ),
-		shouldDisplay: React.PropTypes.func
+		shouldDisplay: React.PropTypes.func,
+		site: React.PropTypes.object,
```

Comment:
Would you mind adding values to `defaultProps` for the new props? For `translate` something like [`_.identity`](https://lodash.com/docs/4.17.2#identity) works.